aptos-move/e2e-testsuite/src/tests/peer_to_peer.rs

```diff
@@ -179,12 +175,16 @@ fn few_peer_to_peer_with_event() {
 
             // check events
             for event in txn_output.events() {
-                if let Ok(payload) = SentEvent::try_from(event) {
+                if let Ok(payload) = WithdrawEvent::try_from(event) {
+                    if payload.amount() == 0 {
```

Comment:
This is because we're also generating withdrawal event for gas burns and in these test setups, gas is 0. This test specifically checks for actual transfers.

Reply:
Updated now that gas burns no longer emit withdrawal events.